bin/next-dev

```diff
@@ -61,6 +61,10 @@ srv.start(argv.port)
   }
 })
 .catch((err) => {
-  console.error(err)
-  process.exit(1)
+  if (err.code === 'EADDRINUSE') {
+    console.error(`Port ${argv.port} is already in use.\nUse \`npm run dev -- -p <some other port>\`.`)
```

Comment:
We are not 100% sure, the user will have `dev` NPM script for next. So, we should try to find it.
Find a NPM script with just `next` but not with `next start` or `next build`.

If we found it, do add that message which is `npm run <script name> -- -p <some other port>`.
Otherwise, just don't display that part.